IdeventAPI/Managers/CompanyManager.cs

```diff
@@ -28,11 +28,20 @@ public List<CompanyModel> GetAll()
             return companies;
         }
 
+        private Func<CompanyModel, AddressModel, AddressModel, CompanyModel> mapping = (companyModel, addressModel, InvoiceModel) =>
+        {
+            companyModel.Address = addressModel;
+            companyModel.InvoiceAddress = InvoiceModel;
+
+            return companyModel;
+        };
+   
+
         public CompanyModel GetById(int id)
         {
             string sql = "EXECUTE spGetCompanyById @companyId";
-            CompanyModel result = _dbConnection.QuerySingle<CompanyModel>(sql, new {companyId = id});
-            return result;
+            var result = _dbConnection.Query(sql, mapping, new {companyId = id}).AsList();
```

Comment:
Hvorfor ændre metode? hvor er mapping variabelen. Eventuelt brug .Single ved slutningen af .Query()

Reply:
Kunne ikke få den til at hente mapping med single metoden. Kunne os se i dappers eksempler at de brugte denne måde. Måske er der flere måder at gøre det på. men det lykkes kun for mig på denne måde pt